egs/ami/asr1/run.sh

```diff
@@ -248,20 +254,18 @@ if [ ${stage} -le 4 ]; then
     for rtask in ${recog_set}; do
     (
         decode_dir=decode_${rtask}_beam${beam_size}_e${recog_model}_p${penalty}_len${minlenratio}-${maxlenratio}
-        
+        feat_recog_dir=${dumpdir}/${rtask}/delta${do_delta}
+
         # split data
         data=data/${rtask}
         split_data.sh --per-utt ${data} ${nj};
         sdata=${data}/split${nj}utt;
 
-        # feature extraction
-        feats="ark,s,cs:apply-cmvn --norm-vars=true data/${train_set}/cmvn.ark scp:${sdata}/JOB/feats.scp ark:- |"
-        if ${do_delta}; then
-        feats="$feats add-deltas ark:- ark:- |"
-        fi
-
         # make json labels for recognition
-        data2json.sh --feat ${data}/feats.scp ${data} ${dict} > ${data}/data.json
+        for j in `seq 1 ${nj}`; do
+            data2json.sh --feat ${feat_recog_dir}/feats.scp \
+                ${sdata}/${j} ${dict} > ${sdata}/${j}/data.json
```

Comment:
This may overwrite ${sdata}/${j}/data.json wrongly when we perform decoding at the same time, but the original one potentially has the same issue, and I did not observe it. So, we can keep it.